drivers/backup/backup.go

```diff
@@ -252,6 +252,9 @@ type Backup interface {
 	WaitForBackupCompletion(ctx context.Context, backupName string, orgID string,
 		timeout time.Duration, timeBeforeRetry time.Duration) error
 
+	WaitForBackupPartialCompletion(ctx context.Context, backupName string, orgID string,
```

Comment:
please add a comment here for what this function is used for. Similar to other functions